cmd/otelcol/Dockerfile

```diff
@@ -37,6 +37,16 @@ RUN if [ "$ARCH" = "amd64" ]; then \
     fi
 RUN find /usr/lib/splunk-otel-collector/agent-bundle -name "*.key" -delete -or -name "*.pem" -delete
 
+FROM debian:10.11 as journal
```

Comment:
Should we use `debian:10` or maybe try `debian:11`?

Reply:
@jcheng-splunk @dmitryax Thanks guys! 

We can surely update it to more latest versions. Image was updated to `10.11` based on this [comment](https://github.com/signalfx/splunk-otel-collector/pull/715#pullrequestreview-779115517) at the time of update.

Also, 10.11 is the latest and stable release for Debian 10 - https://www.debian.org/News/2021/2021100902 

Let me know if we still want to try to update it to 11.x (This will require the entire testing cycle to be repeated as I am not sure of all the changes between Debian 10 and 11) 
Thanks!

Reply:
We should use the latest stable to avoid migrating soon. If 11 is the latest stable version, we should use it. I would pin minor version tho. Dependabot will help with keeping up to date